back/app/policies/user_policy.rb

```diff
@@ -103,17 +103,23 @@ def view_private_attributes?
     !!((user && (instance&.id == user.id || user.admin?)) || instance&.invite_pending?)
   end
 
-  def permitted_attributes
-    shared = [:first_name, :last_name, :email, :password, :avatar, :locale, { onboarding: [:topics_and_areas], custom_field_values: allowed_custom_field_keys, bio_multiloc: CL2_SUPPORTED_LOCALES }]
-    admin? ? shared + role_permitted_params : shared
+  def permitted_attributes_for_create
```

Comment:
@jamesspeake In the end not allowing the update of the email was already implemented in the user model. I put back the code in the policy, as it turns out we can make the distinction between on create and on update in the policy as well (and another controller was using permitted_attrbutes from the policy)